pkg/stub/handler.go

```diff
@@ -25,12 +29,15 @@ type Handler struct {
 }
 
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
-	if event.Deleted {
-		return nil
-	}
 	switch o := event.Object.(type) {
 	case *ingressv1alpha1.ClusterIngress:
-		return h.syncIngressUpdate(o)
+		if event.Deleted {
+			logrus.Infof("Deleting ClusterIngress object: %s", o.Name)
```

Comment:
glog? I'm fine with logrus, we just need to be consistent everywhere.

Reply:
logrus seems to be the logger that the sdk uses